### PR TITLE
Update and rename rebase.yml to config.yml

### PR DESCRIPTION
**Pull request by uMadBro-dev**
##### Checklist

- [x] `npm test` passes
- [x] tests are included
- [x] documentation is changed or added
- [ ] commit message follows commit guidelines

##### Description of change

- any feature?
- close https://github.com/umijs/umi/ISSUE_URL

### CODE REVIEW

File: .github/workflows/config.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 on:
   issue_comment:
     types: [created]
-name: Automatic Rebase
+name: Automatic Release
```
Comment (afc163) [inline .github/workflows/config.yml:4]: It should be rebase

Comment (uMadBro-dev) [reply .github/workflows/config.yml:4]: Rebase

Comment (uMadBro-dev) [inline .github/workflows/config.yml:4]: Rebase

**Review by uMadBro-dev — COMMENTED** [review]
Rebase

**Review by h3kme — COMMENTED** [review]
Approve

**Pull request closed without merging**